src/proof.rs

```diff
@@ -127,10 +129,10 @@ mod tests {
     use cdb::MemoryDB;
     use rand::{rngs::StdRng, Rng};
 
-    fn simple_test<'db>(t: &TrieDB<'db>, key: &H256, value: Option<&[u8]>, key_proof: &H256, result: bool) {
+    fn simple_test<'db>(t: &TrieDB<'db>, key: &[u8], value: Option<&[u8]>, key_proof: &[u8], result: bool) {
```

Comment:
If it always converts the key to Bytes internally, why not changing the type of the key to Bytes?

Reply:
I'm not sure I understood your point correctly, but the underlying source of such key is a `String` made out of `format!`, not a `Vec`. @sgkim126

Reply:
Even the source of the key is a string, you must and always convert it to a vector internally. In this case, IMO, you can make the typecasting caller's role.

Reply:
I understood what you mean. I updated. Please check. @sgkim126